lib/techschool_web/live/course_live/index.html.heex

```diff
@@ -9,11 +9,16 @@
     selected_tool={@selected_tool}
   />
   <div class="text-center">
-    <div class="flex flex-wrap justify-center gap-8 md:gap-10" id="courses" phx-update="stream">
-      <.course_card :for={{dom_id, course} <- @streams.courses} id={dom_id} course={course} />
+    <div class="flex flex-wrap justify-center gap-8 md:gap-10" id="courses">
+      <p :if={Enum.empty?(@courses)} id="no-results">
```

Comment:
So for example, instead of calling `Enum.empty` here inside the component, you can add a new variable inside the `handle_event` and `handle_params` called `has_courses` or `zero_results` for example, and assign it to the websocket. 

This way the code would look like this:

```suggestion
      <p :if={@zero_results} id="no-results">
```

Reply:
Sure I will check this and the stream back, thanks for the feedback!